cowprotocol/cowswap
# feat(permit): refactor permit caching

# Summary

Part of #3146

Built on top/waterfalls onto https://github.com/cowprotocol/cowswap/pull/3164

Refactor permit caching from localStorage to a jotai atom.
Split storage between static (used for quotes with a fixed PK) and user (signed by users before placing the orders).

Is the basis to implement https://github.com/cowprotocol/cowswap/issues/3146, coming next.

# To Test

1. Place SWAP order with permit
2. Place LIMIT order with permit
* Should work as before
